src/index.html

```diff
@@ -4,14 +4,146 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content=
+      "width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="personalInformation block">
+        <legend>Personal information</legend>
+        <label class="input__block">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
[style guide](https://mate-academy.github.io/style-guides/htmlcss.html) 😉